dumper/app/src/main/resources/oracle-stats/awr/hist-cmd-types-awr.sql

```diff
@@ -0,0 +1,46 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  A.con_id "ConId",
+  B.command_type "CommandType",
+  to_char(C.begin_interval_time, 'hh24') "Hour",
+  D.name "CommandName",
+  count(1) "Count",
+  sum(buffer_gets_delta) "SumBufferGets",
+  sum(elapsed_time_delta) "SumElapsedTime",
+  sum(rows_processed_delta) "SumRowsProcessed",
+  sum(executions_delta) "SumExecutions",
```

Comment:
I think we are missing here a few columns that are present in the statspack version: e.g. `disk_reads` , `direct_writes_`

Reply:
The statspack hist-cmd-types includes a join with `stats$SQL_SUMMARY`, which isn't included for awr in the DMA original.
I initially included these rows for statspack, because they are present in the DMA query. However, they are not present in the final output. I'm planning to remove these columns from the statspack version in the future.